packaging_automation/update_package_properties.py

```diff
@@ -353,4 +334,5 @@ def update_all_changes(github_token: non_empty(non_blank(str)), package_properti
         run(f'git push --set-upstream origin {pr_branch}')
         create_pr(arguments.gh_token, pr_branch, commit_message, REPO_OWNER, PROJECT_NAME,
                   project.value.packaging_branch)
+    if not arguments.pipeline and not arguments.is_test:
```

Comment:
To prevent the error below I added this check
https://github.com/citusdata/packaging/runs/3609157981?check_suite_focus=true